src/main/java/org/kainos/ea/api/SalesService.java

```diff
@@ -0,0 +1,26 @@
+package org.kainos.ea.api;
+
+import org.kainos.ea.cli.SalesEmployeeRequest;
+import org.kainos.ea.client.FailedToCreateSalesEmployeeException;
+import org.kainos.ea.cli.SalesEmployee;
+import org.kainos.ea.client.InvalidSalesEmployeeException;
+import org.kainos.ea.db.SalesDAO;
+
+import java.sql.SQLException;
+
+public class SalesService {
+    private SalesDAO salesDao = new SalesDAO();
+    public int createSalesEmployee(SalesEmployeeRequest salesEmployee) throws FailedToCreateSalesEmployeeException {
+        try {
+            int id = salesDao.createSalesEmployee(salesEmployee);
```

Comment:
should we validate the request object before storing in the db?

Reply:
channges made